packages/next/src/client/components/react-dev-overlay/server/middleware-turbopack.ts

```diff
@@ -169,17 +170,23 @@ export function getSourceMapMiddleware(project: Project, distDir: string) {
       return badRequest(res)
     }
 
-    if (filename.startsWith('webpack://next/')) {
+    if (
+      filename.startsWith('webpack://') ||
```

Comment:
aside: Why can't we find sourcemaps for these URLs?

Reply:
Because we don't have a running webpack compiler that we could ask. So these URLs are for prebuilt webpack bundles (from Next.js or third parties) whose originating JS bundles we don't know when this source map is requested. Do you have an idea how we could retrieve them? I guess for the Next.js server bundles we could make them identifiable and mappable, but we want to ignore-list those anyways, right?

Reply:
`node:module#findSourceMap` should be able to. Though I haven't fully grasped how the built-in version works.

Reply:
Oh right, let me check...